components/normalizer/Cargo.toml

```diff
@@ -60,7 +61,14 @@ utf16_iter = ["dep:utf16_iter", "write16"]
 # For dealing with potentially ill-formed UTF8 strings
 utf8_iter = ["dep:utf8_iter"]
 
+# Statically assumes fast-mode trie data
+# unstable_fast_trie_only = ["dep:likely_polyfill"]
+
 [[bench]]
 name = "bench"
 harness = false
 required-features = ["utf16_iter", "utf8_iter"]
+
+# Lint to avoid complaining about unstable_fast_trie_only for now
+[lints.rust]
+unexpected_cfgs = { level = 'allow' }
```

Comment:
```suggestion
unexpected_cfgs = { level = "warn", check-cfg = ['cfg(icu4x_unstable_fast_trie_only)'] }
```

Reply:
This doesn't work. With this suggestion, the lint emits a bunch of warnings.

Reply:
Indeed, that was the problem.